app/controllers/collections_controller.rb

```diff
@@ -46,7 +46,8 @@ def create
                                       access: @collection_form.access,
                                       doi_option: @collection_form.doi_option,
                                       user: current_user,
-                                      deposit_state_event: 'deposit_persist')
+                                      deposit_state_event: 'deposit_persist',
+                                      review_enabled: @collection_form.review_enabled)
```

Comment:
Since this is also happening in the deposit job does it also need to be here?

Reply:
Nope - apparently doesn't need to be. Fixed.